Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -243,4 +246,11 @@ def Runner(env):
             # Tested same FDs on QEMU 6 and 7, not observing the same.
             ret = 0
 
+        if os.name == 'nt' and qemu_version[0] >= '8':
+            # Restore the console mode for Windows on QEMU v8+.
+            std_handle.SetConsoleMode(console_mode)
+        elif os.name != 'nt':
+            # Linux version of QEMU will mess with the print if its run failed, let's just restore it anyway
+            utility_functions.RunCmd ('stty', 'sane')

```

Comment:
Please either set `logging_level = logging.DEBUG` or `capture=false` to stop this from spitting out junk to the terminal post build.

Reply:
Updated with `capture=false`